site/content/component/hono-server.md

```diff
@@ -25,13 +25,16 @@ The following table provides an overview of the configuration options the adapte
-| `HONO_SERVER_BIND_ADDRESS`<br>`--hono.server.bindaddress` | yes | `127.0.0.1` | The IP address the Hono server should bind to. By default the adapter binds to the *loopback device* address, i.e. the adapter will **not** be accessible from other hosts. |
 | `HONO_SERVER_CERT_PATH`<br>`--hono.server.certPath` | no | - | The absolute path to the PEM file containing the certificate the Hono server uses for authenticating to clients. Either this or the `HONO_SERVER_KEY_STORE_PATH` option **must** be set to enable TLS secured connections with clients. |
 | `HONO_SERVER_KEY_PATH`<br>`--hono.server.keyPath` | no | - | The absolute path to the PEM file containing the private key the Hono server uses for authenticating to clients. Either this or the `HONO_SERVER_KEY_STORE_PATH` option **must** be set to enable TLS secured connections with clients. |
 | `HONO_SERVER_KEY_STORE_PATH`<br>`--hono.server.keyStorePath` | no | - | The absolute path to the Java key store containing the private key and certificate the Hono server uses for authenticating to clients. Either this or the `HONO_SERVER_KEY_PATH` and `HONO_SERVER_CERT_PATH` options **must** be set to enable TLS secured connections with clients. The key store format can be either `JKS` or `PKCS12` indicated by a `.jks` or `.p12` file suffix. |
 | `HONO_SERVER_KEY_STORE_PASSWORD`<br>`--hono.server.keyStorePassword` | no | - | The password required to read the contents of the key store. |
 | `HONO_SERVER_MAX_INSTANCES`<br>`--hono.server.maxInstances` | no | *#CPU cores* | The number of verticle instances to deploy. If not set, one verticle per processor core is deployed. |
-| `HONO_SERVER_PORT`<br>`--hono.server.port` | yes | `5672` | The port the server should listen on. |
+| `HONO_SERVER_PORT`<br>`--hono.server.port` | no | `5671` | The secure port the server should listen on. <br> The configured port is guaranteed to be always TLS secured. Thus it requires that `HONO_SERVER_KEY_PATH` is set, otherwise this port will NOT be opened. |
+| `HONO_SERVER_BIND_ADDRESS`<br>`--hono.server.bindaddress` | no | `127.0.0.1` | The IP address the Hono server port should bind to. By default the adapter binds to the *loopback device* address, i.e. the adapter will **not** be accessible from other hosts. |
+| `HONO_SERVER_INSECURE_PORT_ENABLED`<br>`--hono.server.insecurePortEnabled` | no | `false` | Allows the Hono server to open an insecure port (not secured by TLS). This allows for a dual port setup of the Hono server (secure and insecure port). |
+| `HONO_SERVER_INSECURE_PORT`<br>`--hono.server.insecurePort` | no | `5672` | The insecure port the Hono server should listen on (if allowed). <br>If only the insecure port shall be opened, do not set `HONO_SERVER_KEY_STORE_PATH` (preventing that the secure port is opened). <br> For dual port configurations it must not be configured to the same port as `HONO_SERVER_PORT`.|
```

Comment:
can we rephrase

> If only the insecure port shall be opened, do not set `HONO_SERVER_KEY_STORE_PATH` (preventing that the secure port is opened).

to something like this

> In order to open an insecure port only, set this property to `true` but do **not** set  `HONO_SERVER_KEY_STORE_PATH`. This will prevent the secure port from being opened.